services/management/grpc/actions_server.go

```diff
@@ -253,6 +253,40 @@ func (s *actionsServer) StartMySQLShowIndexAction(ctx context.Context, req *mana
 	}, nil
 }
 
+func (s *actionsServer) StartPostgreSQLShowCreateTableAction(ctx context.Context, req *managementpb.StartPostgreSQLShowCreateTableActionRequest) (*managementpb.StartPostgreSQLShowCreateTableActionResponse, error) {
```

Comment:
Here is missed comment for public method. 😃The same in the method below `StartPostgreSQLShowIndexAction`.